tera-templates/docsrs/Cargo.toml.example

```diff
@@ -0,0 +1,44 @@
+{# The example Cargo.toml used in about.html #}
+[package]
+name = "test"
+
+[package.metadata.docs.rs]
+
+# Features to pass to Cargo (default: none)
```

Comment:
Hmm, at one point @pietroalbini suggested changing this to `[]` but it never happened. Do you mind doing it now?
```suggestion
# Features to pass to Cargo (default: [])
```